### DISCUSSION TITLE
Custom Scratch Addons email?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [ScratchAddons/ScratchAddons#380] [Discussion]
To my knowledge, most domain buying services also offer custom emails using that domain. (e.g. _support@scratchaddons.com_). A few questions, does our domain allow this, also is it included with buying the domain? If not it is unessecary, but if it's already included than maybe make support@scratchaddons.com and help@scratchaddons.com?

**Comment by jeffalo** [comment]
yeah but i think W_L would need to either self host an email server or pay more for an emailing service. unless w_l's registrar has free email forwarding

**Comment by easrng** [comment]
Forwardemail or improvmx would work

**Comment by WorldLanguages** [comment]
I don't plan to do this, I'm not 100% sure but I think I could do it for free

**Comment by Explosion-Scratch** [comment]
Okay. Thanks! :)
